e2e/accessibility.ts

```diff
@@ -0,0 +1,90 @@
+import puppeteer from 'puppeteer';
+import { AxePuppeteer } from '@axe-core/puppeteer';
+import { createServer } from 'http-server';
+import { AxeResults } from 'axe-core';
+import logSymbols from 'log-symbols';
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+import assert from 'assert';
+
+const DIST_PATH = path.join(__dirname, '..', 'docs', 'dist');
+const INDEX_HTML = path.join(DIST_PATH, 'index.html');
+const PORT = 36981;
+const IS_CI = 'CI' in process.env;
+
+const main = async (): Promise<void> => {
+  assert(
+    fs.existsSync(DIST_PATH) && fs.existsSync(INDEX_HTML),
+    'Missing dist/'
+  );
+
+  const server = createServer({
+    cache: -1,
+    root: DIST_PATH
+  });
```

Comment:
Need to redirect everything back to `/index.html`. May need to pull in a different dep here.